.eslintrc

```diff
@@ -15,6 +15,7 @@
     "no-unused-expressions": 0,
     "no-param-reassign": 0,
     "no-reserved-keys": 0,
-    "space-infix-ops": 0
+    "space-infix-ops": 0,
+    "array-bracket-spacing": [2, "never"]
```

Comment:
Yes - thanks for that. I put this in from [this comment](https://github.com/acdlite/redux-actions/pull/114#discussion_r73225919), thinking it was off, but obviously markdown isn't linted. 👍